app/src/main/java/com/framgia/soundclound/screen/home/HomeFragment.java

```diff
@@ -0,0 +1,41 @@
+package com.framgia.soundclound.screen.home;
+
+
+import android.databinding.DataBindingUtil;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.framgia.soundclound.R;
+import com.framgia.soundclound.data.source.repository.GenresRepository;
```

Comment:
ERROR: Unused import: com.framgia.soundclound.data.source.repository.GenresRepository. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports